Plugins/BuildServerIntegration/JenkinsIntegration/JenkinsAdapter.cs

```diff
@@ -161,11 +161,13 @@ private async Task<ResponseInfo> GetBuildInfoTaskAsync(string projectUrl, bool f
                     {
                         try
                         {
-                            if (j["lastBuild"] != null && j["lastBuild"]["timestamp"] != null)
+                            if (j["lastBuild"] == null || j["lastBuild"]["timestamp"] == null)
```

Comment:
```suggestion
                            if (j["lastBuild"] is null || j["lastBuild"]["timestamp"] is null)
```

Reply:
Do you want to use the obscure c# 8 "is { }" too?
Forcepushing is null for now

Reply:
In C# 9.0 there's "not is null", we can wait for it :)